README.md

```diff
@@ -1 +1,256 @@
-# IEEE Hackathon Workshop 1
+# IEEE Hackathon Workshop - Designing a REST API with Ballerina
+
+## Introduction
+
+This workshop will guide you through the process of creating a Ballerina REST API that will allow
+you to create, read, update, and delete social media posts.
+
+The Social media REST service exposes the following resources:
+
+| Resource                       | Description                                        |
+|--------------------------------|----------------------------------------------------|
+| `GET api/posts`                | Get all the posts                                  |
+| `GET api/posts/{id}`           | Get the post specified by the id                   |
+| `POST api/post`                | Create a new post                                  |
+| `DELETE api/posts/{id}`        | Delete the post specified by the id                |
+| `GET api/posts/{id}/meta`      | Get the post specified by the id with the metadata |
+
+This workshop has two sessions. In the first session, you will create a Ballerina social media
+service which will use an in-memory table to store the posts. In the second session, you will
+integrate a H2 database to store the posts.
+
+## Areas covered
+
+- Ballerina HTTP service
+- Ballerina HTTP resource
+- HTTP methods
+- Path and query parameters
+- Status code responses
+- Request payload binding
+- Data mapper
+- Ballerina HTTP client
+- HTTP generated client from OpenAPI specification
+- Debugging the Ballerina service
+- Integrating database
+
+## Prerequisites
+
+- Install the latest version(SwanLake Update 10) of Ballerina
```

Comment:
```suggestion
- Install the latest version (SwanLake Update 10) of Ballerina
```
Let's add the link to the downloads page.

Reply:
Fixed via [3f3c7c1](https://github.com/TharmiganK/ieee-hackathon-workshop/pull/1/commits/3f3c7c180536a85a934662716e2d0380bb2f345b)